src/main.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { App } from './app/App';
 import './globals.css';
+import './index.css';
```

Comment:
There is global.css file. I think you should put the properties for fonts and general styles from index.css to global.css